wrappers/src/shared_realm_cs.cpp

```diff
@@ -259,3 +259,120 @@ REALM_EXPORT void shared_realm_write_copy(SharedRealm* realm, uint16_t* path, si
 }
     
 }
+#pragma mark - Creating objects
+
+inline const ObjectSchema& find_schema(const SharedRealm& realm, const Table& table)
+{
+    const StringData object_name(ObjectStore::object_type_for_table_name(table.get_name()));
+    return *realm->schema().find(object_name);
+}
+
+namespace realm
+{
+// specialize overload
+template <>
+size_t Table::find_first(size_t column_ndx, null) const
```

Comment:
It looks like it's in core 3.0.0-rc2 at least, but sync hasn't switched over to that so I guess bindings can't take advantage of it.